tiatoolbox/tools/stainextract.py

```diff
@@ -259,6 +266,14 @@ def __init__(
         regularizer: float = 0.1,
     ) -> None:
         """Initialize :class:`VahadaneExtractor`."""
+        # Issue a warning about the algorithm's stability
+        logger.warning(
```

Comment:
@mostafajahanifar Please can you add links to alternatives as discussed during the meeting?